lib/Net/BGP/Peer.pm

```diff
@@ -352,6 +356,14 @@ sub is_established
     return ( $this->transport->is_established );
 }
 
+sub userdata
```

Comment:
I'm still unconvinced of the necessity of expanding the API, but in any case, I prefer `opaque_ref` to `userdata`.